chrome-launcher/manual-chrome-launcher.js

```diff
@@ -22,11 +22,9 @@ if (args.length) {
   startingUrl = args.find(flag => !flag.startsWith('--'));
 }
 
-const {ChromeLauncher} = require('./chrome-launcher');
-const chromeInstance = new ChromeLauncher({
+const {launch} = require('./chrome-launcher');
+
+launch({
   startingUrl,
   chromeFlags,
 });
```

Comment:
let's add a little `.then(console.log);` at the end here so we know what port we're on